app/config/mq-config.js

```diff
@@ -12,6 +12,12 @@ const mqSchema = joi.object({
     address: joi.string().default('payment'),
     username: joi.string(),
     password: joi.string()
+  },
+  eventTopic: {
+    name: joi.string().default('ffc-pay-event'),
+    address: joi.string().default('payment'),
```

Comment:
I'd remove the default.   This causes errors when local env vars are setup.  It prevents failing fast.

Reply:
removed